src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -78,6 +78,13 @@
      */
     protected $withCount = [];
 
+    /**
+     * The relationship aggregates that should be eager loaded on every query.
+     *
+     * @var array
+     */
+    protected $withAggregates = [];
```

Comment:
Another option would be to remove `withCount` – it's a bigger breaking change, but I think might make more sense, so we don't leave two ways of doing the same thing – feels like we run the risk of more edge cases leaving it this way

Reply:
I would be against removing `withCount` without proper deprecation to give people at least 1 release cycle to work it out slowly. It's a relevant change that could impact packages, tutorials, documentations, a lot of content overall, not just people's application.

Reply:
I would add a deprecation notice, that seems like the only sensible way forward.